src/main/kotlin/org/abimon/eternalJukebox/data/database/HikariDatabase.kt

```diff
@@ -608,6 +608,8 @@ abstract class HikariDatabase : IDatabase {
                         insert.executeBatch()
                     }
                 }
+                 
+                 delay(TIME_BETWEEN_UPDATES_MS)
```

Comment:
This code will currently do nothing; it needs to be a step above (so within the `while (isActive)` loop). Also, use SHORT_ID_UPDATE_TIME_MS rather than TIME_BETWEEN_UPDATES_MS, but after that I'm happy to merge

Reply:
Hi! Thanks for the quick reply!

> This code will currently do nothing; it needs to be a step above (so within the `while (isActive)` loop).

Doh.. I've committed the fix for this.

> Also, use SHORT_ID_UPDATE_TIME_MS rather than TIME_BETWEEN_UPDATES_MS, but after that I'm happy to merge

I've made this change also. The TIME_BETWEEN_UPDATES_MS seemed consistent with the other dispatchers but I'm not going to question it after getting my brackets in a mess :)